test/quantization/core/test_quantized_op.py

```diff
@@ -829,6 +829,44 @@ def test_qadd_relu_same_qparams(self):
             self.assertEqual(qCrelu_hat, qCrelu_out_hat,
                              msg="AddReLU.out failed")
 
+    """Tests the correctness of the cudnn add and add_relu op
+    (Similar to test_qadd_relu_different_qparams, will probably merge in the future)"""
+    @unittest.skipIf(not TEST_CUDNN, "cudnn is not enabled.")
+    @unittest.skip("Local only - currently the qconv2d_cudnn op is bulid "
```

Comment:
what is the effort to make this a flag accessible in pytorch, like TEST_CUDNN? I feel ideally we don't want to copy paste the code for every operator